Gemfile

```diff
@@ -56,9 +56,9 @@ end
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  gem 'capybara', '2.4.4'
```

Comment:
was there a reason that you changed this? if everything works it should be ok but im seeing some travis CI changes failing